### ISSUE TITLE
Creating unassigned work item

### ISSUE THREAD

**Issue by mattzech** [danhellem/github-actions-issue-to-work-item#97]
Is there any way to have the created User Story be an unassigned story? My team is struggling with this because we do not want to have all of the Github issues be assigned to one user who has their inbox flooded with Azure messages every time there is a new issue in the repo. Just checking to see if there is any way around this. Thanks

**Comment by danhellem** [comment]
@mattzech you get this figured out?

**Comment by mattzech** [comment]
Yes, we're working on creating a service account in Azure to get the PAT and use that in the workflow. I'll keep you updated
